public/pages/Monitors/components/MonitorControls/MonitorControls.js

```diff
@@ -47,9 +47,6 @@ const MonitorControls = ({
       <EuiCompressedSelect options={states} value={state} onChange={onStateChange} />
     </EuiFlexItem>
     {monitorActions && <EuiFlexItem grow={false}>{monitorActions}</EuiFlexItem>}
-    <EuiFlexItem grow={false} style={{ justifyContent: 'center' }}>
```

Comment:
should this be shown behind the `useUpdatedUX` condition check?

Reply:
I don't think we need this in MonitorControls at all. I think there was originally duplication in the pagination. We implemented pagination at the bottom here: https://github.com/opensearch-project/alerting-dashboards-plugin/blob/4776c962fc37877400bd56b8cbb74cedff0d02dc/public/pages/Monitors/containers/Monitors/Monitors.js#L514